pkg/certmanager/certmanager.go

```diff
@@ -84,29 +83,22 @@ func (c *CertManager) EnsureCertificate(vol *csiapi.MetaData, keyBundle *util.Ke
 		return nil, err
 	}
 
-	// Write metadata to file
-	metaDataBytes, err := json.Marshal(vol)
-	if err != nil {
-		return nil, err
-	}
-
-	metaPath := filepath.Join(vol.Path, csiapi.MetaDataFileName)
-	if err := ioutil.WriteFile(metaPath, metaDataBytes, 0600); err != nil {
+	if err := util.WriteMetaDataFile(vol); err != nil {
 		return nil, err
 	}
 
-	glog.V(4).Infof("cert-manager: metadata written to file %s", metaPath)
+	glog.V(4).Infof("cert-manager: metadata written to file %s", filepath.Join(vol.Path, csiapi.MetaDataFileName))
```

Comment:
The logged path here is a duplicate of the path used inside `WriteMetaDataFile`. What about introducing a return parameter on `WriteMetaDataFile` to avoid the risk of logging the wrong path?